control-plane/modules/aws-k3s/files/rancher-install.sh

```diff
@@ -52,7 +52,7 @@ spec:
 %{ endif ~}
     extraEnv:
     - name: CATTLE_PROMETHEUS_METRICS
-      value: 'true'
+      value: '${cattle_prometheus_metrics}'
```

Comment:
I don't recall if these variables as text get automatically converted to boolean by Terraform, if that's the case we're good.

Reply:
> I don't recall if these variables as text get automatically converted to boolean by Terraform, if that's the case we're good.

This actually is more about how the rancher helm chart parses env vars. AFAIK it treats all env var values as strings and double-quotes don't seem to be parsed properly for boolean values so we are using single quotes here.